fec.config.js

```diff
@@ -1,27 +1,56 @@
-const { resolve } = require('path');
+const path = require("path");
+const { dependencies, insights } = require("./package.json");
+
+const moduleName = insights.appname.replace(/-(\w)/g, (_, match) =>
+    match.toUpperCase()
+);
+const srcDir = path.resolve(__dirname, "./src");
 
 module.exports = {
-    appUrl: '/insights/patch',
-    debug: true,
-    useProxy: process.env.PROXY === 'true',
-    proxyVerbose: true,
-    plugins: [],
-    ...(process.env.HOT ? { hotReload: process.env.HOT === 'true' } : { hotReload: true }),
+    appName: moduleName,
+    appUrl: "/insights/patch",
+    useProxy: true,
+    ...(process.env.HOT
+        ? { hotReload: process.env.HOT === "true" }
+        : { hotReload: true }),
     ...(process.env.port ? { port: parseInt(process.env.port) } : {}),
     moduleFederation: {
+        moduleName,
+        exposes: {
+            "./RootApp": path.resolve(__dirname, "./src/AppEntry"),
+            "./SystemDetail": path.resolve(__dirname, "./src/index.js"),
+        },
         shared: [
             {
-                'react-router-dom': {
+                "react-router-dom": {
                     singleton: true,
                     import: false,
-                    version: '^6.8.1',
-                    requiredVersion: '>=6.0.0 <7.0.0'
-                }
-            }
+                    version: dependencies["react-router-dom"],
+                    requiredVersion: ">=6.0.0 <7.0.0",
+                },
+            },
+            {
+                "@unleash/proxy-client-react": {
```

Comment:
This shouldn't be needed, and might actually cause issues. Chrome is already sharing its instance of the unleash client.

Reply:
Good call, this and the url below are left-overs from a copy of code 🫤.

This isn't ready for a review.